src/Raven.Server/Documents/Indexes/MapReduce/ReduceMapResultsBase.cs

```diff
@@ -127,8 +128,8 @@ protected ReduceMapResultsBase(T indexDefinition, IndexStorage indexStorage, Met
                     {
                         var message = $"Failed to execute reduce function for reduce key '{modifiedState.Tree.Name}' on a leaf page #{page} of '{_indexDefinition.Name}' index.";
 
-                        if (Log.IsWarnEnabled)
-                            Log.WarnException(message, e);
+                        if (_logger.IsOperationsEnabled)
+                            _logger.Operations(message, e);
```

Comment:
This isn't an operator level error. It is already reported in the indexes error, and should go to the regular log